radicle-node/Cargo.toml

```diff
@@ -7,14 +7,19 @@ edition = "2021"
 
 [features]
 test = ["radicle/test", "radicle-crypto/test", "qcheck"]
+new_arch = ["poll-reactor", "netservices"]
```

Comment:
What about implementing it directly into the project in place of making it an optional feature?

Reply:
It will be, but it's easier to review this way.